app/models/pitch.rb

```diff
@@ -11,7 +11,12 @@ class Pitch < ApplicationRecord
   validates :pitch_type, :hour_price, presence: true,
   numericality: {only_integer: true}
 
-  scope :sort_list_pitch, ->{order :name}
+  scope :search_by_price_range, lambda {|max_price|
+    return if max_price.blank?
+
+    where("hour_price BETWEEN 0 AND ?", max_price.to_s)
+  }
+  scope :search, ->(name){where("name LIKE ?", name).order :name}
```

Comment:
```suggestion
  scope :search_by_name, ->(name){where("name LIKE ?", name)}
```

sao lại có order trong này 
`.order :name`